### PR TITLE
Switch my qt packages to qt-specific mkDerivation

### PR DESCRIPTION
**Pull request by fgaz**
###### Motivation for this change

Just learned about it in #63330 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/sfxr-qt/default.nix [hunk 1]
```diff
@@ -1,11 +1,13 @@
-{ stdenv, fetchFromGitHub
+{ lib
+, mkDerivation
+, fetchFromGitHub
 , cmake
 , qtbase, qtquickcontrols2
```
Comment (worldofpeace) [inline pkgs/applications/audio/sfxr-qt/default.nix]: can you do this one per line too?

Comment (fgaz) [reply pkgs/applications/audio/sfxr-qt/default.nix]: Done

**Review by worldofpeace — APPROVED** [review]
LGTM, checked the binaries were wrapped.

**Comment by worldofpeace** [comment]
Lol :smile: I kinda enjoyed testing `sfxt-qt` a bit too much.

**Pull request merged**